docs/tutorial/tutorial_03.rst

```diff
@@ -78,3 +78,29 @@ Now supposing your access token value is `123456` you can try to access your aut
 ::
 
     curl -H "Authorization: Bearer 123456" -X GET http://localhost:8000/secret
+
+Working with Rest_framework generic class based views
+-----------------------------------------------------
+
+If you have completed the `Django REST framework tutorial
+<https://www.django-rest-framework.org/tutorial/3-class-based-views/#using-generic-class-based-views>`_,
+you will be familiar with the 'Snippet' example, in particular the SnippetList and SnippetDetail classes.
+
+It would be really nice to reuse those views, but also support token handling. Instead of reworking
+those classes to be ProtectedResourceView based, the solution is much simpler than that.
+
+Assuming you have already modified the settings as was already shown.
+
+The key is setting a class variable to override the default *permissions_classes* with something that will use our :term:`Access Token` properly.
+
+.. code-block:: python
+
+    from django.contrib.auth.decorators import login_required
+
+    class SnippetList(generics.ListCreateAPIView):
+        ...
+        permission_classes = [TokenHasReadWriteScope]
```

Comment:
I believe this will override other default permission classes. Perhaps you can override the  `get_permission_classes` method

Reply:
Yes - it overrides the default classes. And maybe there is a better way to do this, but it was the way that worked for me. 
Overriding  get_permission_classes would basically do the same thing with more code - unless you need more complicated logic. Again - open to suggestions.

Reply:
I added another paragraph that hopefully resolves your concern. There is so many ways to "solve" the issue of a newbie who starts with the Snippets tutorial. I like the "simple/fast" solution presented, but now have references additional information that will guide the reader.